apps/mobile/src/app/(home)/account/[account]/activity.tsx

```diff
@@ -0,0 +1,72 @@
+import { useAccountAddresses } from '@/hooks/use-account-addresses';
+import { useAccountActivityQuery } from '@/queries/activity/account-activity.query';
+import { Account, deserializeAccountId } from '@/store/accounts/accounts';
+import dayjs from 'dayjs';
+import { useLocalSearchParams } from 'expo-router';
+
+import { ActivityLayout } from '../../activity/activity.layout';
+import { ActivityCell } from '../../activity/components/activity-cell';
+import { ActivityIcon } from '../../activity/components/activity-icon';
+import { formatActivityType } from '../../activity/utils/format-activity';
+import { isTransfer } from '../../activity/utils/is-send-or-receive';
+import { configureAccountParamsSchema } from './index';
+
+export default function AccountActivityScreen() {
+  const params = useLocalSearchParams();
+  const { accountId } = configureAccountParamsSchema.parse(params);
+  const { fingerprint, accountIndex } = deserializeAccountId(accountId);
+  const accountAddresses = useAccountAddresses(fingerprint, accountIndex);
+  const { data: activity, isLoading } = useAccountActivityQuery(accountAddresses);
+
+  if (isLoading) {
+    return null;
+  }
+  return (
+    <ActivityLayout>
+      {activity?.map((activity, index) => {
```

Comment:
There seem to be two variations of this list, would it make sense to extract it into a single configurable ActivityList component that encapsulates the selection logic, virtualization, etc?